### ISSUE TITLE
SacCer3 GWAS colors appear to be inconsistent

### ISSUE THREAD

**Issue by paul-shannon** [igvteam/igv.js#1573]
Hi Jim,
So far we see this chromosome color inconsistency only in yeast.
Small html file copied in below, ought to reproduce this for you: the same track is loaded twice with a different track name.    igv.js 2.13.8. 

![image](https://user-images.githubusercontent.com/2480712/211397111-d3ab1ed9-3257-43aa-8ca0-72505052914d.png)

```
<html lang="en">
<head>
  <meta charset="utf-8">
  <meta name="viewport" content="width=device-width, initial-scale=1">
  <title>GWAS</title>
  <script src='http://code.jquery.com/jquery-1.12.4.min.js'></script>
  <script src='http://code.jquery.com/ui/1.12.1/jquery-ui.min.js'></script>
  <script src='http://maxcdn.bootstrapcdn.com/bootstrap/3.3.7/js/bootstrap.min.js'></script>
  <script src="https://cdn.jsdelivr.net/npm/igv@2.13.8/dist/igv.min.js"></script>
</head>

<body>

<div id="igvDiv" style="padding-top: 50px;padding-bottom: 20px; height: auto"></div>
<script type="text/javascript">
  var browser;

 $(document).ready(function () {
    var igvDiv, options;
    igvDiv = $("#igvDiv")[0];
     options = {locus: "all",
               minimumBases: 5,
               showRuler: true,
               genome: "sacCer3",
               tracks:[
                 {type: "gwas",
                  format: "gwas",
                  name: "1) yeast V",
                  url: "https://igv-data.systemsbiology.net/testFiles/gwas/tbl.gwas.yeast.chrV.tsv",
                  indexed: false,
                  columns: {
                      chromosome: 1,
                      position: 2,
                      value: 3
                  }
                 },
                 {type: "gwas",
                  format: "gwas",
                  name: "2) yeast V",
                  url: "https://igv-data.systemsbiology.net/testFiles/gwas/tbl.gwas.yeast.chrV.tsv",
                  indexed: false,
                  columns: {
                      chromosome: 1,
                      position: 2,
                      value: 3
                  }
                 }                   
               ]
              };
        browser = igv.createBrowser(igvDiv, options);
    });

</script>
</body>
</html>



```

**Comment by paul-shannon** [comment]
Hi Jim,

Thanks - now I understand.  An optional color table would be very helpful.

 - Paul

**Comment by jrobinso** [comment]
This is fixed, in version 1.12.7.  The specified colors in the table above are now valid for roman numeral chromosome names, with the exception of "10 / X".  This would conflict with the existing chromosome "X".   So "X" will always get color "rgb(204, 153, 0)".

 Also,  I added the ability to specify a custom color table as shown below, but you shouldn't need to use it for this case.  Unless you want to use the color for chr "10" for "X".


```
                 type: "gwas",
                format: "gwas",
                name: "GWAS - custom color table",
                colorTable: {
                    "1": "red",
                    "2": "blue",
                    "3": "green"
                },
                url: "https://s3.amazonaws.com/igv.org.demo/gwas_sample.gwas.gz",
                indexed: false
```

**Comment by paul-shannon** [comment]
Thank you, Jim!
